.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: ci
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build and Run Tests with Gradle
+        run: ./gradlew build
```

Comment:
Will be the `./gradlew build` also run tests? 
It's not apparent from the command name. Or is this a GithubAction thing?

Reply:
Yeah, it does.